Dockerfile

```diff
@@ -52,7 +52,9 @@ COPY ./ /code/
 
 RUN python manage.py collectstatic --noinput
 
+ARG GIT_TAG=
 ARG GIT_COMMIT=
```

Comment:
`git describe` is definitely better than the full commit hash. I don't have any feelings about GIT_COMMIT, it just seemed potentially redundant, and if it's only used for display (or not actually given a value?) could be replaced with GIT_TAG. :man_shrugging: